The job `mingw-check` of your PR [failed](https://dev.azure.com/rust-lang/rust/_build/results?buildId=31919&view=logs&jobId=74054d28-c774-5fab-ecc2-352fe71b230e) ([pretty log](https://rust-lang.github.io/rust-log-analyzer/log-viewer/#https://dev.azure.com/rust-lang/e71b0ddf-dd27-435a-873c-e30f86eea377/_apis/build/builds/31919/logs/67), [raw log](https://dev.azure.com/rust-lang/e71b0ddf-dd27-435a-873c-e30f86eea377/_apis/build/builds/31919/logs/67)). Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.

<details><summary><i>Click to expand the log.</i></summary>

```plain
##[section]Starting: Linux mingw-check
##[section]Starting: Initialize job
Agent name: 'Azure Pipelines 2'
Agent machine name: 'fv-az619'
Current agent version: '2.170.1'
##[group]Operating System
16.04.6
LTS
LTS
##[endgroup]
##[group]Virtual Environment
Environment: ubuntu-16.04
Version: 20200604.1
Included Software: https://github.com/actions/virtual-environments/blob/ubuntu16/20200604.1/images/linux/Ubuntu1604-README.md
##[endgroup]
Agent running as: 'vsts'
Prepare build directory.
Set build variables.
Download all required tasks.
Download all required tasks.
Downloading task: Bash (3.163.3)
Checking job knob settings.
   Knob: AgentToolsDirectory = /opt/hostedtoolcache Source: ${AGENT_TOOLSDIRECTORY} 
   Knob: AgentPerflog = /home/vsts/perflog Source: ${VSTS_AGENT_PERFLOG} 
Start tracking orphan processes.
##[section]Finishing: Initialize job
##[section]Starting: Configure Job Name
==============================================================================
---
========================== Starting Command Output ===========================
[command]/bin/bash --noprofile --norc /home/vsts/work/_temp/26f61cda-129e-40f9-9666-cb6395f8edfd.sh

##[section]Finishing: Disable git automatic line ending conversion
##[section]Starting: Checkout rust-lang/rust@refs/pull/73331/merge to s
Task         : Get sources
Description  : Get sources from a repository. Supports Git, TfsVC, and SVN repositories.
Version      : 1.0.0
Author       : Microsoft
---
##[command]git remote add origin https://github.com/rust-lang/rust
##[command]git config gc.auto 0
##[command]git config --get-all http.https://github.com/rust-lang/rust.extraheader
##[command]git config --get-all http.proxy
##[command]git -c http.extraheader="AUTHORIZATION: basic ***" fetch --force --tags --prune --progress --no-recurse-submodules --depth=2 origin +refs/heads/*:refs/remotes/origin/* +refs/pull/73331/merge:refs/remotes/pull/73331/merge
---
 ---> f883e675ad62
Step 6/7 : ENV RUN_CHECK_WITH_PARALLEL_QUERIES 1
 ---> Using cache
 ---> c0b156eb069c
Step 7/7 : ENV SCRIPT python3 ../x.py test src/tools/expand-yaml-anchors &&            python3 ../x.py check --target=i686-pc-windows-gnu --host=i686-pc-windows-gnu &&            python3 ../x.py build --stage 0 src/tools/build-manifest &&            python3 ../x.py test --stage 0 src/tools/compiletest &&            python3 ../x.py test src/tools/tidy &&            python3 ../x.py doc --stage 0 src/libstd &&            /scripts/validate-toolstate.sh
 ---> 8541bab6b38c
Successfully built 8541bab6b38c
Successfully tagged rust-ci:latest
Built container sha256:8541bab6b38c07f1b7eb787539b9cbe93daa6ac4458d3d7bd8a8921622a14ba1
---
    Checking rustc_span v0.0.0 (/checkout/src/librustc_span)
    Checking rustc_errors v0.0.0 (/checkout/src/librustc_errors)
    Checking rustc_feature v0.0.0 (/checkout/src/librustc_feature)
    Checking rustc_parse_format v0.0.0 (/checkout/src/librustc_parse_format)
    Checking chalk-rust-ir v0.10.0
    Checking rustc_query_system v0.0.0 (/checkout/src/librustc_query_system)
    Checking rustc_hir v0.0.0 (/checkout/src/librustc_hir)
    Checking rustc_session v0.0.0 (/checkout/src/librustc_session)
    Checking chalk-solve v0.10.0
    Checking rustc_parse v0.0.0 (/checkout/src/librustc_parse)
---
configure: llvm.assertions      := True
configure: rust.codegen-units-std := 1
configure: rust.verify-llvm-ir  := True
configure: rust.debug-assertions := True
configure: build.configure-args := ['--enable-sccache', '--disable-manage-submodu ...
configure: writing `config.toml` in current directory
configure: 
configure: run `python /checkout/x.py --help`
configure: 
---
Hugepagesize:       2048 kB
DirectMap4k:      112576 kB
DirectMap2M:     4081664 kB
DirectMap1G:     5242880 kB
+ python3 ../x.py test src/tools/expand-yaml-anchors
Ensuring the YAML anchors in the GitHub Actions config were expanded
Ensuring the YAML anchors in the GitHub Actions config were expanded
Building stage0 tool expand-yaml-anchors (x86_64-unknown-linux-gnu)
   Compiling unicode-xid v0.2.0
   Compiling syn v1.0.11
   Compiling linked-hash-map v0.5.2
   Compiling lazy_static v1.4.0
   Compiling lazy_static v1.4.0
   Compiling yaml-rust v0.4.3
   Compiling quote v1.0.2
   Compiling thiserror-impl v1.0.5
   Compiling thiserror v1.0.5
   Compiling yaml-merge-keys v0.4.0
   Compiling expand-yaml-anchors v0.1.0 (/checkout/src/tools/expand-yaml-anchors)
Build completed successfully in 0:00:28
+ python3 ../x.py check --target=i686-pc-windows-gnu --host=i686-pc-windows-gnu
    Finished dev [unoptimized] target(s) in 0.16s
Checking rustdoc artifacts (x86_64-unknown-linux-gnu -> i686-pc-windows-gnu)
---
    Checking rustc_span v0.0.0 (/checkout/src/librustc_span)
    Checking rustc_errors v0.0.0 (/checkout/src/librustc_errors)
    Checking rustc_feature v0.0.0 (/checkout/src/librustc_feature)
    Checking rustc_parse_format v0.0.0 (/checkout/src/librustc_parse_format)
    Checking chalk-rust-ir v0.10.0
    Checking rustc_query_system v0.0.0 (/checkout/src/librustc_query_system)
    Checking rustc_hir v0.0.0 (/checkout/src/librustc_hir)
    Checking rustc_session v0.0.0 (/checkout/src/librustc_session)
    Checking chalk-solve v0.10.0
    Checking rustc_parse v0.0.0 (/checkout/src/librustc_parse)
---
 }
 
-
 #[derive(Clone)]
 pub struct TcpStream(Arc<Socket>);
Diff in /checkout/src/libstd/sys/hermit/net.rs at line 158:
Diff in /checkout/src/libstd/sys/hermit/net.rs at line 158:
                     ((addr.0[8] as u16) << 8) | addr.0[9] as u16,
                     ((addr.0[10] as u16) << 8) | addr.0[11] as u16,
                     ((addr.0[12] as u16) << 8) | addr.0[13] as u16,
-                    ((addr.0[14] as u16) << 8) | addr.0[15] as u16)),
+                    ((addr.0[14] as u16) << 8) | addr.0[15] as u16,
+                )),
             ),
             _ => {
Diff in /checkout/src/libstd/sys/hermit/net.rs at line 165:
Diff in /checkout/src/libstd/sys/hermit/net.rs at line 165:
                 return Err(io::Error::new(ErrorKind::Other, "peer_addr failed"));
+            }
         };
 
         Ok(saddr)
         Ok(saddr)
Diff in /checkout/src/libstd/sys/hermit/net.rs at line 249:
                     ((addr.0[8] as u16) << 8) | addr.0[9] as u16,
                     ((addr.0[10] as u16) << 8) | addr.0[11] as u16,
                     ((addr.0[12] as u16) << 8) | addr.0[13] as u16,
-                    ((addr.0[14] as u16) << 8) | addr.0[15] as u16)),
+                    ((addr.0[14] as u16) << 8) | addr.0[15] as u16,
+                )),
             ),
             _ => {
Diff in /checkout/src/libstd/sys/hermit/net.rs at line 256:
Diff in /checkout/src/libstd/sys/hermit/net.rs at line 256:
                 return Err(io::Error::new(ErrorKind::Other, "accept failed"));
+            }
         };
 
 
         Ok((TcpStream(Arc::new(Socket(handle))), saddr))
Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/bin/rustfmt" "--config-path" "/checkout" "--edition" "2018" "--unstable-features" "--skip-children" "--check" "/checkout/src/libstd/sys/hermit/net.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
Build completed unsuccessfully in 0:00:43
== clock drift check ==
  local time: Sat Jun 13 23:07:51 UTC 2020
  network time: Sat, 13 Jun 2020 23:07:51 GMT
  network time: Sat, 13 Jun 2020 23:07:51 GMT
== end clock drift check ==

##[error]Bash exited with code '1'.
##[section]Finishing: Run build
##[section]Starting: Checkout rust-lang/rust@refs/pull/73331/merge to s
Task         : Get sources
Description  : Get sources from a repository. Supports Git, TfsVC, and SVN repositories.
Version      : 1.0.0
Author       : Microsoft
Author       : Microsoft
Help         : [More Information](https://go.microsoft.com/fwlink/?LinkId=798199)
==============================================================================
Cleaning any cached credential from repository: rust-lang/rust (GitHub)
##[section]Finishing: Checkout rust-lang/rust@refs/pull/73331/merge to s
Cleaning up task key
Start cleaning up orphan processes.
Terminate orphan process: pid (3708) (python)
##[section]Finishing: Finalize Job
```

</details><p></p>

[I'm a bot](https://github.com/rust-ops/rust-log-analyzer)! I can only do what humans tell me to, so if this was not helpful or you have suggestions for improvements, please ping or otherwise contact **`@rust-lang/infra`**. ([Feature Requests](https://github.com/rust-ops/rust-log-analyzer/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request))

--- a/src/libstd/sys/hermit/net.rs
+++ b/src/libstd/sys/hermit/net.rs
@@ -149,13 +149,10 @@ impl TcpStream {
                 IpAddr::V4(Ipv4Addr::new(addr.0[0], addr.0[1], addr.0[2], addr.0[3])),
                 port,
             ),
-            Ipv6(ref addr) => SocketAddr::new(
-                IpAddr::V6(Ipv6Addr::new(addr.0)),
-                port,
-            ),
+            Ipv6(ref addr) => SocketAddr::new(IpAddr::V6(Ipv6Addr::new(addr.0)), port),
             _ => {
                 return Err(io::Error::new(ErrorKind::Other, "peer_addr failed"));
-            },
+            }
         };
 
         Ok(saddr)
@@ -232,13 +229,10 @@ impl TcpListener {
                 IpAddr::V4(Ipv4Addr::new(addr.0[0], addr.0[1], addr.0[2], addr.0[3])),
                 port,
             ),
-            Ipv6(ref addr) => SocketAddr::new(
-                IpAddr::V6(Ipv6Addr::new(addr.0)),
-                port,
-            ),
+            Ipv6(ref addr) => SocketAddr::new(IpAddr::V6(Ipv6Addr::new(addr.0)), port),
             _ => {
                 return Err(io::Error::new(ErrorKind::Other, "accept failed"));
-            },
+            }
         };
 
         Ok((TcpStream(Arc::new(Socket(handle))), saddr))

--- a/src/libstd/sys/hermit/net.rs
+++ b/src/libstd/sys/hermit/net.rs
@@ -35,7 +35,6 @@ impl Drop for Socket {
     }
 }
 
-
 #[derive(Clone)]
 pub struct TcpStream(Arc<Socket>);
 

--- a/src/libstd/sys/hermit/net.rs
+++ b/src/libstd/sys/hermit/net.rs
@@ -150,15 +150,7 @@ impl TcpStream {
                 port,
             ),
             Ipv6(ref addr) => SocketAddr::new(
-                IpAddr::V6(Ipv6Addr::new(
-                    ((addr.0[0] as u16) << 8) | addr.0[1] as u16,
-                    ((addr.0[2] as u16) << 8) | addr.0[3] as u16,
-                    ((addr.0[4] as u16) << 8) | addr.0[5] as u16,
-                    ((addr.0[6] as u16) << 8) | addr.0[7] as u16,
-                    ((addr.0[8] as u16) << 8) | addr.0[9] as u16,
-                    ((addr.0[10] as u16) << 8) | addr.0[11] as u16,
-                    ((addr.0[12] as u16) << 8) | addr.0[13] as u16,
-                    ((addr.0[14] as u16) << 8) | addr.0[15] as u16)),
+                IpAddr::V6(Ipv6Addr::new(addr.0)),
                 port,
             ),
             _ => {
@@ -241,15 +233,7 @@ impl TcpListener {
                 port,
             ),
             Ipv6(ref addr) => SocketAddr::new(
-                IpAddr::V6(Ipv6Addr::new(
-                    ((addr.0[0] as u16) << 8) | addr.0[1] as u16,
-                    ((addr.0[2] as u16) << 8) | addr.0[3] as u16,
-                    ((addr.0[4] as u16) << 8) | addr.0[5] as u16,
-                    ((addr.0[6] as u16) << 8) | addr.0[7] as u16,
-                    ((addr.0[8] as u16) << 8) | addr.0[9] as u16,
-                    ((addr.0[10] as u16) << 8) | addr.0[11] as u16,
-                    ((addr.0[12] as u16) << 8) | addr.0[13] as u16,
-                    ((addr.0[14] as u16) << 8) | addr.0[15] as u16)),
+                IpAddr::V6(Ipv6Addr::new(addr.0)),
                 port,
             ),
             _ => {

